tests/new_upgrades/test_errata.py

```diff
@@ -0,0 +1,285 @@
+"""Test for Errata related Upgrade Scenarios
+
+:Requirement: UpgradedSatellite
+
+:CaseAutomation: Automated
+
+:CaseComponent: ErrataManagement
+
+:Team: Phoenix-content
+
+:CaseImportance: Critical
+
+"""
+
+from box import Box
+from fauxfactory import gen_alpha
+import pytest
+from wait_for import wait_for
+
+from robottelo import constants
+from robottelo.config import settings
+from robottelo.hosts import ContentHost
+from robottelo.logging import logger
+from robottelo.utils.shared_resource import SharedResource
+
+
+def _create_custom_rhel_n_client_repos(target_sat, product, v_major):
+    """Create custom RHEL (system) and Satellite Client repos and sync them"""
+    repos = []
+    rhel_repo_urls = [settings.repos[f'rhel{v_major}_os']]
+    if v_major > 7:
+        rhel_repo_urls = [rhel_repo_urls[0]['BASEOS'], rhel_repo_urls[0]['APPSTREAM']]
+
+    client_repo_url = settings.repos.satclient_repo[f'rhel{v_major}']
+    if not all([all(rhel_repo_urls), client_repo_url]):
+        raise ValueError(
+            f'Repo URLs for RHEL {v_major} system repository or/and '
+            f'Satellite Client repository for RHEL {v_major} are not set.'
+        )
+    # Satellite Client repo
+    repos.append(
+        target_sat.api.Repository(
+            product=product.id,
+            name=f'RHEL {v_major} Client',
+            content_type='yum',
+            url=client_repo_url,
+        ).create()
+    )
+    # RHEL system repo(s)
+    for url in rhel_repo_urls:
+        repos.append(
+            target_sat.api.Repository(
+                product=product.id,
+                name=f'RHEL {v_major} {url.split("/")[-4]}',
+                content_type='yum',
+                url=url,
+            ).create()
+        )
+
+    for repo in repos:
+        repo.sync(timeout=3000)
+    return repos
+
+
+def _get_rh_rhel_n_client_repos(target_sat, org, rhel_client):
```

Comment:
Hmm, this looks unused anywhere, but let's keep it since this is just a CP. We would have to initiate the removal through master.